iOS_SDK/OneSignalSDK/Source/OneSignal.m

```diff
@@ -1959,6 +1886,24 @@ + (NSString*)checkForProcessedDups:(NSDictionary*)customDict lastMessageId:(NSSt
     return nil;
 }
 
++ (void)handleWillPresentNotificationInForegroundWithPayload:(NSDictionary *)payload withCompletion:(OSNotificationDisplayTypeResponse)completion {
```

Comment:
same change on https://github.com/OneSignal/OneSignal-iOS-SDK/pull/687/files, shouldn't we make a single Pr with these changes on master then rebase? or is this intended?